fastlane/Scanfile

```diff
@@ -4,12 +4,13 @@
 # In general, you can use the options available
 # fastlane scan --help
 
+project("StreamChat.xcodeproj")
+scheme("StreamChat")
```

Comment:
These should not be in Scanfile since integration tests (Carthage/Cocoapods/SPM) build different project/workspaces (see failed task in CI)

Reply:
Yup, fixed already. Don't worry.